code/game/machinery/computer/apc_control.dm

```diff
@@ -103,7 +139,7 @@
 					auth_id = "[ID.registered_name] ([ID.assignment]):"
 					log_activity("[auth_id] attempted to log into the terminal")
 					playsound(src, 'sound/machines/terminal_error.ogg', 50, FALSE)
-					to_chat(usr, span_warning("ID REJECTED - Access Denied."))
+					to_chat(usr, span_danger("ID REJECTED - Access Denied."))
```

Comment:
Would it be better if the machine uses ``say()`` instead, so this would also technically be balloon alert via runechat?